components/html-block/html-block.js

```diff
@@ -230,10 +240,21 @@ class HtmlBlock extends RtlMixin(LitElement) {
 		`;
 	}
 
-	updated(changedProperties) {
+	async updated(changedProperties) {
 		super.updated(changedProperties);
+		if (changedProperties.has('html') && this.html !== undefined && this.html !== null && !this._hasSlottedContent) {
+			const renderContainer = this.shadowRoot.querySelector('.d2l-html-block-rendered');
```

Comment:
Looks like these 3 lines could be factored out (since they are duplicated) into an appropriately named method.

Reply:
Oops, you're right.